packages/http-client-csharp/emitter/src/index.ts

```diff
@@ -2,7 +2,8 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 export { configurationFileName, tspOutputFileName } from "./constants.js";
-export { $onEmit } from "./emitter.js";
+export { $onEmit, CSharpEmitterContext, transformJSONProperties } from "./emitter.js";
+export { createModel } from "./lib/client-model-builder.js";
```

Comment:
so we could just exporting this method, without exporting its return type.
after we do this, on autorest.csharp side, when they call `createModel`, what type they would get as a return?

Reply:
We still get a CodeModel - I guess the export is implicit.